lib/src/int64.dart

```diff
@@ -70,18 +70,18 @@ class Int64 implements IntX {
       throw FormatException("No digits in '$s'");
     }
 
-    int d0 = 0, d1 = 0, d2 = 0; //  low, middle, high components.
+    var d0 = 0, d1 = 0, d2 = 0; //  low, middle, high components.
```

Comment:
Please leave `int` everywhere in this file. This code is performance critical and has been broken before by well-meaning changes that accidentally generalize the types, resulting in large regressions of protobuf parsing.